lib/game.rb

```diff
@@ -12,6 +12,6 @@ def run(output)
   private
 
   def empty_board
-    Board.new.stringify
+    Board.new.grid
```

Comment:
hm, so are you suggesting? 
```
def to_s
  grid
end 
```
in Board? This was a bit of what I was talking about in our above conversation https://github.com/swetagsanghavi/tic-tac-oo/pull/7#discussion_r397964627 where it felt like I was re-writing attr_reader by doing something like this. What do you think?

Reply:
Yep, that example is what I had in mind. I wouldn't consider that to be re-writing `attr_reader`, because it has the complexity of knowing that `grid` is a string that's suitable for external callers to use as a string. I'd only consider a method as re-writing `attr_reader` if the method name and the variable name were the same.

Reply:
True true true. We're using the method name for external callers to understand what they are receiving. Thanks for the clarification!

Reply:
> external callers to understand what they are receiving

Yep! It all goes back to focusing on the messages (aka method names) that are being passed around.

Reply:
I like that frame. Will keep reminding myself of that!